### DISCUSSION TITLE
월드 빌드 실패 요인이 궁금합니다.

### DISCUSSION THREAD

**Discussion by peullea** [naverz/zepeto-studio-kor#898] [5. Q&A]
동일한 계정으로 로그인한 상태고, 유니티 상에서는 잘 작동합니다.
서버도 키고(Publish)옆에 있는거도 켜놓고 하는데도 그러네요.
할 수 있는 방법은 다 해본거 같은데 뭐가 문제인지 알 수 있는 방법이 있을까요?
![asdf](https://user-images.githubusercontent.com/102412179/174430141-fb806449-1d2b-4591-942c-303a72047932.PNG)
위는 패키지 파일로 보낸 후의 로그입니다.

**Comment by x4cf** [comment]
경로에 한글이 있는것으로 보입니다. 경로에 한글을 제거하고 다시 시도해보는게 좋을것같아요 :)

**Reply by x4cf** [reply]
파란색으로 가린부분이 한글인가요?

**Reply by peullea** [reply]
한글이 아닌걸로 알고있습니다만 한번 바꿔서 해보겠습니다

**Comment by YSHFM** [comment]
android sdk도 확인해보시는 것도 좋을 것 같습니다. 유니티 개발 시에 안드로이드 sdk 버전이 안맞거나 해서 빌드 안되는 경우를 많이 봅니다
